event/transform.go

```diff
@@ -27,12 +27,18 @@ const (
 	// ETTransformDatasetPreview is an abbreviated dataset document in a transform
 	// Payload will be a *dataset.Dataset Preview
 	ETTransformDatasetPreview = Type("tf:DatasetPreview")
+
+	// ETTransformWriteRun is for when we have written the results of a
+	// transform to the logbook
+	// Payload will be a dsref.VersionInfo
+	ETTransformWriteRun = Type("tf:WriteRun")
```

Comment:
This should be a logbook event, not a transform event.

Reply:
So glad you said this, I meant to ask this/highlight this on self-review. We didn't have a list of logbook events yet, so put it here by "default". Glad it was egregious enough to get a mention anyway, lol